Add info for Redis Binding create operation (#2539)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-bindings/redis.md

- `create`

> We can store a record in Redis using the `create` command. This will set a key to hold a value. If a key already holds a value, it will be overwritten.

Reviewer: ```suggestion
You can store a record in Redis using the `create` operation. This sets a key to hold a value. If the key already exists, the value will be overwritten.
```

Author: Thanks! Updated.

### Request

```json
{
  "operation": "create",
  "metadata": {
    "key": "key1"
  },
  "data": {
    "Hello": "World",
    "Lorem": "Ipsum"
  }
}
```

### Response

> An HTTP 204 (No Content) and empty body will be returned if successful.

Reviewer: ```suggestion
An HTTP 204 (No Content) and empty body is returned if successful.
```
